cmake/Modules/CppUTestNormalizeTestOutputLocation.cmake

```diff
@@ -1,10 +0,0 @@
-# Override output properties to put test executable at specificied location
```

Comment:
This created collisions between configs, by using the same output path for all of them:

```sh
cmake --preset MSVC
cmake --build cpputest_build --config Release
cmake --build cpputest_build --config Debug # Overwrites Release outputs
ctest --test-dir cpputest_build -C Release # Actually runs Debug tests
```